src/nacl/signing.py

```diff
@@ -56,10 +58,18 @@ class VerifyKey(encoding.Encodable, StringFixer, object):
     signatures.
 
     :param key: [:class:`bytes`] Serialized Ed25519 public key
-    :param encoder: A class that is able to decode the `key`
+    :param encoder: (Deprecated) A class that is able to decode the `key`
     """
 
-    def __init__(self, key, encoder=encoding.RawEncoder):
+    def __init__(self, key, encoder=None):
+        if encoder is None:
+            encoder = encoding.RawEncoder
+        else:
+            warn(("Implicit encoding/decoding of signing keys "
```

Comment:
We should just say encoding/decoding of signing keys is deprecated. Remove the words explicit and implicit from this msg.